management/server/user_test.go

```diff
@@ -550,3 +541,115 @@ func TestUser_GetUsersFromAccount_ForUser(t *testing.T) {
 	assert.Equal(t, 1, len(users))
 	assert.Equal(t, mockServiceUserID, users[0].ID)
 }
+
+func TestDefaultAccountManager_SaveUser(t *testing.T) {
+	manager, err := createManager(t)
+	if err != nil {
+		t.Fatal(err)
+		return
+	}
+
+	regularUserID := "regularUser"
+
+	tt := []struct {
+		name           string
+		adminUserID    string
+		regularUserID  string
```

Comment:
`adminUserID` and `regularUserId` have no influence on the test so can be removed form test case and hardcoded via constants down below

Reply:
makes sense